config/migrations/20210726-government-domains.ttl

```diff
@@ -0,0 +1,39 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
```

Comment:
Er moet ook een `.graph` file toegevoegd worden om te zorgen dat deze data ingeladen wordt in de public graph. Zie ook https://github.com/mu-semtech/mu-migrations-service#inserting-data-in-a-specific-graph-using-turtle-files-experimental

Een voorbeeld hiervan vind je in https://github.com/kanselarij-vlaanderen/app-persberichten/blob/master/config/migrations/20210628123600-mock-login-users.graph.

Zelfde opmerking geldt voor de andere migration file in deze PR.